api/src/main/java/keywhiz/api/automation/v2/SecretContentsResponseV2.java

```diff
@@ -0,0 +1,47 @@
+package keywhiz.api.automation.v2;
```

Comment:
It seemed to me that having separate fields for secrets which were successfully read (and their contents) and secrets which were not successfully read (and any error messages thrown in the reading) would be easier than returning a single map of secrets to "the results of the operation" and making the client distinguish success and failure. 

At the moment, the only error case is that a secret was not found, so `errorSecrets` could be a list rather than a map; however, having it as a map leaves flexibility to introduce other error cases later on without changing the API.